build_projects/shared-build-targets-utils/Utils/SharedFrameworkPublisher.cs

```diff
@@ -234,15 +241,8 @@ private void GenerateRuntimeGraph(DotNetCli dotnetCli, string destinationDeps)
             }
         }
 
-        private void CopyHostArtifactsToSharedFramework(string sharedFrameworkNameAndVersionRoot, string hostFxrVersion)
+        private void CopyHostArtifactsToSharedFramework(string sharedFrameworkNameAndVersionRoot)
         {
-            File.Copy(
-                Path.Combine(_corehostLockedDirectory, HostArtifactNames.DotnetHostBaseName),
```

Comment:
dotnet.exe from a specific SharedFX version was used to run a _portable_ app against that version of the framework and not a standalone app. I am not aware of any scenario requiring this and, for 2.0, it is fine to remove it. However, we should keep an eye out to understand if anyone is impacted with this breaking change.